### DISCUSSION TITLE
Plex - Tracks - Different Artist?

### DISCUSSION THREAD

**Discussion by RJ-Make** [music-assistant/support#2801] [Q&A] [answered]
Not sure if something has changed, and I need to adjust a setting. I'm seeing tracks from other artist, when I select an artist.

Here is one example:

![2024-08-24_9-07-05](https://github.com/user-attachments/assets/75cde4f1-3511-4855-84f7-98fe02a0da54)

**Comment by OzGav** [comment]
You don’t say what version you are on?

**Reply by RJ-Make** [reply]
2.2.2 
2024.8.1

**Reply by OzGav** [reply, marked as answer]
This is fixed in the beta. Please wait for the next stable version.
